### PR TITLE
🚫 No more surveyor submissions.. thus no more API v1

### PR DESCRIPTION
**Pull request by rowanseymour**
These have been going thru mailroom for a while now. Before merging this I'll push a new surveyor release which removes the option to submit legacy submissions to RapidPro.

### CODE REVIEW

**Comment by nicpottier** [comment]
One consideration here is other deployments. They are likely not running the new surveyor.. so if we rip this out before they ship they are going to have stranded submissions. So we may not want to merge this until we upmerge and mention this in the release notes.

**Comment by rowanseymour** [comment]
We can wait til after the upmerge.. tho I have a hard time believing anyone else is running their own fork of Surveyor.

**Comment by nicpottier** [comment]
Bet there's a few in the Ihlasoft world.

File: temba/settings_common.py [hunk 1]
```diff
@@ -953,9 +953,6 @@
     context["brand"] = dict(slug=brand["slug"], styles=brand["styles"])
     COMPRESS_OFFLINE_CONTEXT.append(context)
 
-MAGE_API_URL = "http://localhost:8026/api/v1"
```
Comment (rowanseymour) [inline temba/settings_common.py]: unrelated - just found this searching for "v1"

File: temba/api/v1/serializers.py [hunk 2]
```diff
@@ -114,29 +80,17 @@ def __init__(self, **kwargs):
...
-
-    pass
-
-
-class WriteSerializer(serializers.Serializer):
-    """
-    The normal REST framework way is to have the view decide if it's an update on existing instance or a create for a
-    new instance. Since our logic for that gets relatively complex, we have the serializer make that call.
-    """
+class MsgCreateSerializer(serializers.Serializer):  # pragma: no cover
```
Comment (rowanseymour) [inline temba/api/v1/serializers.py]: left for now as it's used by webhooks but it'll be removed in next pr

**Comment by rowanseymour** [comment]
Chatted with @nicpottier about this and we're going to merge this and just warn people when we do the upmerge that this is one of the features which has gone now. It's on them to hold back if the happen to have their own surveyor fork still making legacy submissions.

**Pull request merged**